### PR TITLE
https://issues.redhat.com/browse/ACM-7779 bindingOverrides 2.9

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-7779

### CODE REVIEW

File: governance/policy_overview.adoc [hunk 1]
```diff
@@ -63,6 +63,9 @@ spec:
 apiVersion: apps.open-cluster-management.io/v1
 kind: PlacementBinding
 metadata:
+bindingOverrides:
+  remediationAction: [null|enforce]
+subFilter: [false|true]
```
Comment (yiraeChristineKim) [inline governance/policy_overview.adoc]: Sorry I just check crd, it was change to  -restricted
```suggestion
subFilter: [restricted | null]
```

File: governance/policy_overview.adoc [hunk 2]
```diff
@@ -110,6 +113,18 @@ spec:
 
 *Note:* You can view policy violations based on the standards and categories that you define for your policy on the _Policies_ page, from the console.
 
+//is this description accurate? 
+//What does null mean for the user in this case
+//bindingOverrides and subFilter are these new or did I miss these in a previous release? If these are new, we need to also add a what's new entry
+| `bindingOverrides.remediationAction`
+| Optional
+| When this parameter is set to `enforce`, it provides a way for you to control the remediation action of the related `PlacementBinding` resources for configuration policies. The default value is `null`.
```
Comment (yiraeChristineKim) [inline governance/policy_overview.adoc]: I think it is better `contorol the remediation ` ->`override the remediation action` but either way make sense

File: governance/policy_overview.adoc [hunk 3]
```diff
@@ -110,6 +113,18 @@ spec:
...
+
+| `subFilter`
+| Optional
+| Use this parameter to enable filtering for the `bindingOverrides` parameter.
+//is subfilter a subsection of bindingOverrides?
```
Comment (yiraeChristineKim) [inline governance/policy_overview.adoc]: yes it is related to bindingoverrides. In crd,  it is explained `This field provides the ability to select a subset of bound`

File: governance/policy_overview.adoc [hunk 4]
```diff
@@ -230,6 +245,9 @@ apiVersion: policy.open-cluster-management.io/v1
 kind: PlacementBinding 
 metadata: 
   name: binding-policy-role
+bindingOverrides:
+  remediationAction: null
+subFilter: false
```
Comment (yiraeChristineKim) [inline governance/policy_overview.adoc]: null
```suggestion
subFilter: null
```

File: release_notes/whats_new.adoc [hunk 5]
```diff
@@ -59,6 +59,9 @@ For other Application topics, see link:../applications/app_management_overview.a
 //2.9:6709
 * You can use the `InformOnly` parameter to check the status of Kubernetes objects without enforcing a configuration policy, even if the parent policy `remediationAction` is set to `enforce`. See link:../governance/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller] to learn more.
 
+//2.9:1384
+* Control the remediation action for your configuration policies by using the `bindingOverrides` parameter. For more information, see link:../governance/policy_overview.adoc#policy-yaml-table[Policy YAML table].
```
Comment (yiraeChristineKim) [inline release_notes/whats_new.adoc]:
```suggestion
* Overrides the remediation action for your bound policies by using the `bindingOverrides` parameter. For more information, see link:../governance/policy_overview.adoc#policy-yaml-table[Policy YAML table]. 
```
Not only configuration policy, iam cert also applied

File: governance/policy_overview.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/policy_overview.adoc]: It seems odd and confusing to list two values for the parameters. we can explain the values in the table , and I realize that the intention for the sample is to show an empty policy. Just a way for the user to see the outline of the policy, so i think we should remove the values here

File: governance/policy_overview.adoc (hunk 2 above)
Comment (dockerymick) [inline governance/policy_overview.adoc]:
```suggestion
| When this parameter is set to `enforce`, it provides a way for you to override the remediation action of the related `PlacementBinding` resources for configuration policies. The default value is `null`.
```

File: governance/policy_overview.adoc (hunk 1 above)
Comment (dockerymick) [inline governance/policy_overview.adoc]:
```suggestion
  remediationAction:
subFilter:
```

File: governance/policy_overview.adoc [hunk 6]
```diff
@@ -110,6 +113,18 @@ spec:
...
+//is this description accurate? 
+//What does null mean for the user in this case
+//bindingOverrides and subFilter are these new or did I miss these in a previous release? If these are new, we need to also add a what's new entry
+| `bindingOverrides.remediationAction`
+| Optional
+| When this parameter is set to `enforce`, it provides a way for you to override the remediation action of the related `PlacementBinding` resources for configuration policies. The default value is `null`.
+
+| `subFilter`
+| Optional
+| Use this parameter to enable filtering for the `bindingOverrides` parameter.
```
Comment (dockerymick) [inline governance/policy_overview.adoc]:
```suggestion
| Set this parameter to `restriction` to select a subset of bound policies. The default value is `null`.
```

File: governance/policy_overview.adoc [hunk 7]
```diff
@@ -110,6 +113,18 @@ spec:
...
+//is subfilter a subsection of bindingOverrides?
```
Comment (dockerymick) [inline governance/policy_overview.adoc]:
```suggestion
```

File: governance/policy_overview.adoc (hunk 6 above)
Comment (dockerymick) [reply governance/policy_overview.adoc]: @yiraeChristineKim thoughts on this suggestion?

Comment (yiraeChristineKim) [reply governance/policy_overview.adoc]: Great

File: governance/policy_overview.adoc [hunk 8]
```diff
@@ -110,6 +113,17 @@ spec:
 
 *Note:* You can view policy violations based on the standards and categories that you define for your policy on the _Policies_ page, from the console.
 
+//is this description accurate? 
+//What does null mean for the user in this case
+//bindingOverrides and subFilter are these new or did I miss these in a previous release? If these are new, we need to also add a what's new entry
```
Comment (dockerymick) [inline governance/policy_overview.adoc]:
```suggestion
```

File: governance/policy_overview.adoc [hunk 9]
```diff
@@ -230,6 +241,9 @@ apiVersion: policy.open-cluster-management.io/v1
 kind: PlacementBinding 
 metadata: 
   name: binding-policy-role
+bindingOverrides:
+  remediationAction: null
+subFilter: null
```
Comment (dockerymick) [inline governance/policy_overview.adoc:246]: Should the YAML be like this? Based on the What's new entry, it seems. like subFilter is a part of the parameter section
```suggestion
bindingOverrides:
  remediationAction: null
  subFilter: null
```

Comment (yiraeChristineKim) [reply governance/policy_overview.adoc:246]: nono 
```
bindingOverrides:
  remediationAction: null
subFilter: null
```
This is correct

**Review by yiraeChristineKim — APPROVED** [review]

File: release_notes/whats_new.adoc [hunk 10]
```diff
@@ -59,6 +59,9 @@ For other Application topics, see link:../applications/app_management_overview.a
 //2.9:6709
 * You can use the `InformOnly` parameter to check the status of Kubernetes objects without enforcing a configuration policy, even if the parent policy `remediationAction` is set to `enforce`. See link:../governance/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller] to learn more.
 
+//2.9:1384
+* Overrides the remediation action for your bound policies by using the `bindingOverrides` parameter. For more information, see link:../governance/policy_overview.adoc#policy-yaml-table[Policy YAML table].
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]:
```suggestion
* Overrides the remediation action for your bound policies by using the `bindingOverrides` parameter. You can also use the `subFilter` parameter to select the bound policies. For more information, see link:../governance/policy_overview.adoc#policy-yaml-table[Policy YAML table]. 
```

**Review by oafischer — APPROVED** [review]
Looks great, thanks!

**Comment by dockerymick** [comment]
@yiraeChristineKim @oafischer thanks for the reviews and approvals. Merging this PR

**Pull request merged**